core/src/main/java/de/danoeh/antennapod/core/storage/DBTasks.java

```diff
@@ -199,11 +199,12 @@ private static void refreshFeeds(final Context context,
                 } catch (DownloadRequestException e) {
                     e.printStackTrace();
                     DBWriter.addDownloadStatus(
-                            new DownloadStatus(feed, feed
-                                    .getHumanReadableIdentifier(),
-                                    DownloadError.ERROR_REQUEST_ERROR, false, e
-                                    .getMessage()
-                            )
+                            new DownloadStatus(feed,
+                                               feed.getHumanReadableIdentifier(),
+                                               DownloadError.ERROR_REQUEST_ERROR,
+                                               false,
+                                               e.getMessage(),
+                                               false)
```

Comment:
Users can also press the refresh button. I think this method might be called when users press the refresh button, so it is not always initiated by the system. Not sure what to pick here, though.

Reply:
Good catch, managed to get it sorted out.